client_test.go

```diff
@@ -0,0 +1,164 @@
+package helmclient
+
+import (
+	"helm.sh/helm/v3/pkg/repo"
+	"k8s.io/client-go/rest"
+)
+
+func ExampleNew() {
+	opt := &Options{
+		RepositoryCache:  "/tmp/.helmcache",
+		RepositoryConfig: "/tmp/.helmrepo",
+		Debug:            true,
+		Linting:          true,
+	}
+
+	// Construct a new Helm client, where '_' is the constructed client.
+	// Change this empty assignment to get access to the client's services.
+	_, err := New(opt)
```

Comment:
I'm assuming this is because the compiler would complain if you assigned the client to a variable, without actually using it?

Reply:
Exactly! Maybe doing an empty assignment like below would make this example more practical. What are your thoughts on this?

```go
helmClient, err := New(opt)
_ = helmClient
```